server/utils/assessments/utils.ts

```diff
@@ -530,6 +530,14 @@ const caseNotesFromAssessment = (assessment: Assessment) =>
 const acctAlertsFromAssessment = (assessment: Assessment) =>
   assessment.application?.data?.['prison-information']?.['case-notes']?.acctAlerts || []
 
+const rejectionRationaleFromAssessmentResponses = (assessment: Assessment): string => {
+  const response = getResponseForPage(assessment, 'make-a-decision', 'make-a-decision')?.Decision || ''
+
+  if (Array.isArray(response)) return ''
```

Comment:
Why would this return an array?

Reply:
For the `make-a-decision` page it will always return a string but TS doesn't know that, `getResponsesForPage` sometimes returns an array.
Rather than cast it as a string which could cause issues down the line I think it's better to guard against it being an array to keep TS happy

Reply:
Ah, nice 👍